IsraelHiking.Web/sources/application/components/dialogs/shareDialog.html

```diff
@@ -21,16 +19,17 @@
                     <textarea mdInput rows="2" type="text" [(ngModel)]="description" placeholder="{{resources.description}}" fxFill></textarea>
                 </md-input-container>
             </div>
+            <div class="input-wrapper-spacer"></div>
             <div fxLayout="row">
                 <button md-raised-button color="primary" (click)="generateUrl()" [disabled]="isLoading" fxFill angulartics2On="click" angularticsCategory="Share" angularticsEvent="Share create">{{resources.generateUrlToShareWithYourFriends}}</button>
             </div>
         </div>
         <div [hidden]="!shareAddress">
-            <div fxLayout="row" fxLayoutAlign="space-around none">
-                <div fxFlex="33">
+            <div [fxLayout]="direction" fxLayout.xs="column" fxLayout.md="row" fxLayoutAlign="space-around none">
```

Comment:
direction is not a variable, what did you mean here.
Try using `ng build --prod` and you'll get the relevant error message.